modyn/utils/utils.py

```diff
@@ -136,6 +136,25 @@ def package_available_and_can_be_imported(package: str) -> bool:
         return False
 
 
+def instantiate_class(class_module_name: str, class_name: str, *class_params: Any) -> Any:
```

Comment:
thanks for creating this! could you also add one test to the utils test file? that would be amazing. thank you!

Reply:
Added some tests

Reply:
thx